CONTRIBUTING.md

```diff
@@ -1,118 +1,182 @@
+[Join today!](http://chat.poliastro.space/)
 
-Besides, the [wiki](https://github.com/poliastro/poliastro/wiki) is open
-for everybody to edit, so feel free to add new content.
+### Improve the Documentation
 
-To build the docs, you must first create a development environment (see
-below) and then in the `docs/` directory run:
-```bash
-    $ cd docs
-    $ make html
-```
-After this, the new docs will be inside `build/html`. You can open them
-by running an HTTP server:
-```bash
-    $ cd build/html
-    $ python -m http.server
-    Serving HTTP on 0.0.0.0 port 8000 ...
-```
-And point your browser to <http://0.0.0.0:8000>.
+Documentation can always be expanded and improved.
+The docs are stored in text files under the `docs/source` directory,
+and the Python classes and methods also feature inline docs
+in the form of docstrings.
 
-## Small scripts
+### Contribute your research scripts
 
 We would love to give your Astrodynamics scripts a home!
 Please head to [our `contrib/` directory](https://github.com/poliastro/poliastro/tree/main/contrib)
 for further information.
 
-## Code writing
+### Fix bugs and add new features
 
 Code contributions are welcome! If you are looking for a place to start,
-help us fixing bugs in poliastro and check out the [\"good-first-issue\" tag](https://github.com/poliastro/poliastro/labels/good%20first%20issue).
-Those should be easier to fix than the others and require less knowledge
-about the library.
+check out the ["good-first-issue" label](https://github.com/poliastro/poliastro/labels/good%20first%20issue)
+on our issue tracker. Those tasks should be easier to fix than the others
+and require less knowledge about the library.
+
+## How to contribute
 
-If you are hesitant on what IDE or editor to use, just choose one that
-you find comfortable and stick to it while you are learning. People have
-strong opinions on which editor is better so I recommend you to ignore
-the crowd for the time being - again, choose one that you like :)
+### Work from GitHub
 
-If you ask me for a recommendation, I would suggest PyCharm (complete
-IDE, free and gratis, RAM-hungry) or vim (powerful editor, very
-lightweight, steep learning curve). Other people use Spyder, emacs,
-gedit, Notepad++, Sublime, Atom\...
+GitHub makes it very easy to make small contributions
+directly from your browser, without having to install any additional software.
+To get familiar with the process, you can do
+[this interactive GitHub training](https://lab.github.com/githubtraining/introduction-to-github).
+Once you have finished it, you can edit the poliastro source files
+straight from GitHub and open pull requests from your browser.
 
-You will also need to understand how git works. git is a decentralized
+### Work locally
+
+For more complex contributions, it will be more effective
+to set up a local development environment and work from your own computer.
+
+The most important thing is to understand how git works. git is a decentralized
```

Comment:
Does the "g" in "git" generally written uppercase, i.e. Git instead of git?

Reply:
You're right, in the official site they call it `Git` https://git-scm.com/ Will change it